admin_guide/table_management.rst

```diff
@@ -1,6 +1,54 @@
 Table Management
 $$$$$$$$$$$$$$$$$$
 
+.. _table_size:
+
+Determining Table and Relation Size
+###################################
+
+The usual way to find table sizes in PostgreSQL, :code:`pg_total_relation_size`, drastically under-reports the size of distributed tables. All this function does on a Citus cluster is reveal the size of tables on the coordinator node. In reality the data in distributed tables lives on the worker nodes (in shards), not on the coordinator. A true measure of distributed table size is obtained as a sum of shard sizes. Citus provides helper functions to query this information.
+
++------------------------------------------+----------------------------------------------------------+
+| UDF                                      | Returns                                                  |
++==========================================+==========================================================+
+| citus_relation_size(relation_name)       | Size of actual data in table (the "main fork").          |
+|                                          |                                                          |
+|                                          | A relation can be the name of a table or an index.       |
++------------------------------------------+----------------------------------------------------------+
+| citus_table_size(relation_name)          | citus_relation_size plus                                 |
+|                                          |                                                          |
+|                                          | * size of free space map                                 |
```

Comment:
Could we add links to FSM and Visibility Map definitions or descriptions in Postgres?